modules/aws/centos-gfx/vars.tf

```diff
@@ -100,6 +100,16 @@ variable "admin_ssh_key_name" {
   type        = string
 }
 
+variable "pcoip_agent_location" {
+  description = "URL of Teradici PCoIP Graphics Agent"
+  default     = "https://downloads.teradici.com/rhel/"
```

Comment:
Given how the agent is installed by downloading a repo and its public key, i think for CentOS it make sense to have a pcoip_agent_repo_public_key and default to https://downloads.teradici.com/rhel/teradici.pub.gpg and a pcoip_agent_repo_url and default to https://downloads.teradici.com/rhel/pcoip.repo